src/app.js

```diff
@@ -28,6 +28,27 @@ app.get('/', function (req, res) {
     routes.root(req, res, db);
 });
 
+app.get('/global-stats.html', function (req, res) {
```

Comment:
Up to you, the files returned **are** html files but that's fine. I remove the extensions.